### ISSUE TITLE
problem: users cannot log in

### ISSUE THREAD

**Issue by gazhayes** [Blockrazor/blockrazor#1066]
Problem: when logging in users see a 500 error.

Server logs:  
```
[34mI20180410-22:42:56.181(-4)? [39mException while invoking method 'login' ReferenceError: log is not defined
[34mI20180410-22:42:56.182(-4)? [39m    at imports/api/users/users.js:31:5
[34mI20180410-22:42:56.182(-4)? [39m    at runAndHandleExceptions (packages/callback-hook.js:152:24)
[34mI20180410-22:42:56.182(-4)? [39m    at packages/callback-hook.js:159:12
[34mI20180410-22:42:56.182(-4)? [39m    at packages/accounts-base/accounts_server.js:139:13
[34mI20180410-22:42:56.183(-4)? [39m    at Hook.each (packages/callback-hook.js:128:15)
[34mI20180410-22:42:56.183(-4)? [39m    at AccountsServer.Ap._validateLogin (packages/accounts-base/accounts_server.js:136:27)
[34mI20180410-22:42:56.183(-4)? [39m    at AccountsServer.Ap._attemptLogin (packages/accounts-base/accounts_server.js:342:8)
[34mI20180410-22:42:56.183(-4)? [39m    at DDPCommon.MethodInvocation.methods.login (packages/accounts-base/accounts_server.js:530:21)
[34mI20180410-22:42:56.183(-4)? [39m    at maybeAuditArgumentChecks (packages/ddp-server/livedata_server.js:1768:12)
[34mI20180410-22:42:56.183(-4)? [39m    at DDP._CurrentMethodInvocation.withValue (packages/ddp-server/livedata_server.js:719:19)

```

**Comment by gazhayes** [comment]
Probably broken in #1064 but haven't verified.

**Comment by gazhayes** [comment]
@blockrazorbot claim

**Comment by gazhayes** [comment]
@blockrazorbot claim

**Comment by gazhayes** [comment]
@blockrazorbot add "in progress"

**Comment by AshHimself** [comment]
@gazhayes This is fixed in #1067 but I think @janat08 needs to review as it may of broken in other locations

**Comment by janat08** [comment]
No that's okay, that's how it used to be.
